NDTensors/src/lib/Sectors/src/abstractcategory.jl

```diff
@@ -32,14 +19,100 @@ function Base.show(io::IO, cs::Vector{<:AbstractCategory})
   return nothing
 end
 
+Base.isless(c1::AbstractCategory, c2::AbstractCategory) = isless(label(c1), label(c2))
+
+# =================  Misc  ======================
 function trivial(category_type::Type{<:AbstractCategory})
   return error("`trivial` not defined for type $(category_type).")
 end
 
 istrivial(c::AbstractCategory) = (c == trivial(typeof(c)))
 
-function dual(category_type::Type{<:AbstractCategory})
+# name conflict with LabelledNumber.label. TBD is that an issue?
+label(c::AbstractCategory) = error("method `label` not defined for type $(typeof(c))")
+
+# TBD dimension in Sectors or in GradedAxes namespace?
+function dimension(c::AbstractCategory)
```

Comment:
Is there a more descriptive name we could use for this? `dimension` is a very generic name but here it has a particular meaning.

Reply:
`quantum_dimension` would be best then.

Reply:
That sounds reasonable to me. Is that a common term for it in the literature? @emstoudenmire any opinions on this?

That has the advantage, or disadvantage, of sounding like it is straight out of [Ant-Man](https://en.wikipedia.org/wiki/Ant-Man_(film)).

Reply:
For fusion categories, it is the conventional expression (see e.g. https://physics.stackexchange.com/questions/186348/what-are-the-quantum-dimensions-of-the-primary-fields-for-sun-level-k-kac and the author)

Reply:
This was a point I was wanting to discuss with you both. For an tensor index, there will be two kinds of dimension. One will be the dimension which is the 'number of settings' of that index. So like the usual dimension we will use when making a matrix out of GradedAxes. 

But then there is the "full" dimension which would be the dimension one would find if one got rid of the symmetry and restored the tensor into a dense tensor.

The "quantum dimensions" are the extra factors that restore this full dimension. At least for groups it is mathematically clear that's what they mean.  So for U(1) they are 1 while for SU(2) an irrep J has a quantum dimension of (2J+1).

For categories it's a little bit less clear what the quantum dimension really means in every case. In some cases it is, like for Majorana fermions it's clear. Every time you include a pair of fermions the vector space size doubles, so the quantum dimension is sqrt(2). But in some cases, like tau anyons, the quantum dimension is just the "asymptotic growth rate" of the dimensions of tensors, not the actual dimensions.

Hopefully that's helpful background to decide on good names.

I can't think of anything better than `quantum_dimension` right now other than maybe `full_dimension`? But `quantum_dimension` is indeed a standard term. I suppose we could always add an alias name later if we want to.